src/core/server/saved_objects/export/get_sorted_objects_for_export.test.ts

```diff
@@ -399,6 +393,79 @@ describe('getSortedObjectsForExport()', () => {
     ).rejects.toThrowErrorMatchingInlineSnapshot(`"Can't export more than 1 objects"`);
   });
 
+  test('sorts objects within type', async () => {
+    savedObjectsClient.find.mockResolvedValueOnce({
```

Comment:
There also previously wasn't coverage when objects weren't provided - so this also has the benefit of including that path.